device/alpaka/include/traccc/alpaka/utils/get_device_info.hpp

```diff
@@ -0,0 +1,36 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2024 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+#include <alpaka/acc/Tag.hpp>
+
+namespace traccc::alpaka {
+
+// Get alpaka tag for current device
+#if defined(ALPAKA_ACC_GPU_CUDA_ENABLED)
+using AccTag = ::alpaka::TagGpuCudaRt;
+#elif defined(ALPAKA_ACC_GPU_HIP_ENABLED)
+using AccTag = ::alpaka::TagGpuHipRt;
+#elif defined(ALPAKA_ACC_SYCL_ENABLED)
+#if defined(ALPAKA_SYCL_ONEAPI_CPU)
+using AccTag = ::alpaka::TagCpuSycl;
+#elif defined(ALPAKA_SYCL_ONEAPI_FPGA)
+using AccTag = ::alpaka::TagFpgaSyclIntel;
+#elif defined(ALPAKA_SYCL_ONEAPI_GPU)
+using AccTag = ::alpaka::TagGpuSyclIntel;
+#endif
+#elif defined(ALPAKA_ACC_CPU_B_SEQ_T_THREADS_ENABLED)
+using AccTag = ::alpaka::TagCpuThreads;
+#endif
```

Comment:
I don't like these sitting here. They are not used in the interface of `get_device_info()`. So if the client needs them, they should be put into a header that has a more descriptive name. 🤔

Reply:
`device_properties.hpp`?

Reply:
Sure, that would sound okay.